wsdl/ver10/advancedsecurity/wsdl/advancedsecurity.wsdl

```diff
@@ -10,7 +10,7 @@
 -->
 <wsdl:definitions xmlns:wsdl="http://schemas.xmlsoap.org/wsdl/" xmlns:tas="http://www.onvif.org/ver10/advancedsecurity/wsdl" xmlns:soap="http://schemas.xmlsoap.org/wsdl/soap12/" targetNamespace="http://www.onvif.org/ver10/advancedsecurity/wsdl">
 	<wsdl:types>
-		<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" targetNamespace="http://www.onvif.org/ver10/advancedsecurity/wsdl" version="19.12">
+		<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:tt="http://www.onvif.org/ver10/schema" elementFormDefault="qualified" targetNamespace="http://www.onvif.org/ver10/advancedsecurity/wsdl" version="23.12">
```

Comment:
I tried to generate the code for this WSDL from VS but got an error. Adding the following line resolved the issue. I'm not familiar enough with XML/WSDL if this is the proper way to do things or not
```suggestion
		<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:tt="http://www.onvif.org/ver10/schema" elementFormDefault="qualified" targetNamespace="http://www.onvif.org/ver10/advancedsecurity/wsdl" version="23.12">
			<xs:import namespace="http://www.onvif.org/ver10/schema" schemaLocation="../../../ver10/schema/onvif.xsd"/>
```

Reply:
@jflevesque-genetec can you please provide more details about the problem you are seeing with WS?

Reply:
If you try to load the WSDL in SoapUI for example, you have the following errors:
```
Fri Sep 01 10:30:18 EDT 2023:ERROR:org.apache.xmlbeans.XmlException: C:\Program Files\SmartBear\SoapUI-5.7.1\bin\file:\wsdl\ver10\advancedsecurity\wsdl\advancedsecurity.wsdl:0: error: src-resolve: type 'DeviceEntity@http://www.onvif.org/ver10/schema' not found.
``` 

In VS, it simply gives out an unknown error.

Adding the import to the WSDL resolves the issue.

Reply:
Agree that schema is broken.

The security schema doesn't include onvif.xsd. Instead of including onvif.xsd I suggest to add StringList definition and attribute 'token' to AuthorizationServerConfiguration instead of generating nested complex type including DeviceEntity.